--- posix_ipc-1.2.0.orig/setup.py	2025-04-17 05:54:31.082254090 -0500
+++ posix_ipc-1.2.0/setup.py	2025-04-17 05:55:01.116635392 -0500
@@ -24,8 +24,7 @@
 
 # Linux & FreeBSD require linking against the realtime libs.
 # This causes an error on other platforms
-if "REALTIME_LIB_IS_NEEDED" in system_info:
-    libraries.append("rt")
+libraries.append("rt")
 
 ext_modules = [Extension("posix_ipc",
                          SOURCE_FILES,
